pages/api/memories/[id].js

```diff
@@ -0,0 +1,52 @@
+import dbConnect from "../../../db/dbConnect";
+import Memory from "../../../db/models/Memory";
+
+export default async function handler(req, res) {
+  await dbConnect();
+
+  const { id } = req.query;
+  const { method } = req;
+
+  if (method === "GET") {
+    // Read Memory by ID
+    try {
+      const memory = await Memory.findById(id); // this is a mongoose collection
+      // Display Memory as JSON object in Browser
+      res.status(200).json(memory);
+    } catch (error) {
+      res.status(400).json("Something went wrong", { error });
+      res.status(404).json("ID not found", { error });
+    }
+  }
+
+  if (method === "DELETE") {
+    try {
+      await Memory.findByIdAndDelete(id);
```

Comment:
wo kommt die ID in dem Fall her?? ist das in Z. 68 der _app.js das "+ id"?

Reply:
Positiv, korrekt! 👍